android/src/main/java/com/pspdfkit/flutter/pspdfkit/PspdfkitPlugin.java

```diff
@@ -49,6 +61,12 @@ public void onMethodCall(MethodCall call, Result result) {
             case "present":
                 String documentPath = call.argument("document");
                 requireNotNullNotEmpty(documentPath, "Document path");
+
+                HashMap<String, Object> configurationMap = call.argument("configuration");
+                ConfigurationAdapter configurationAdapter;
```

Comment:
Declaration and assignment should be merged.

Reply:
🙈